### ISSUE TITLE
Package `open-telemetry/opentelemetry-auto-symfony` can not be installed

### ISSUE THREAD

**Issue by gnumoksha** [open-telemetry/opentelemetry-php#1197]
**Describe your environment**
Linux
PHP 8.2.13
composer 2.6.6
opentelemetry module

**Steps to reproduce**
```
echo '{"require": {}}' > composer.json
composer require symfony/flex
composer require open-telemetry/opentelemetry-auto-symfony
```

**What is the expected behavior?**
Installation of `open-telemetry/opentelemetry-auto-symfony` in the latest version

**What is the actual behavior?**
The following error:
```
Cannot use open-telemetry/opentelemetry-auto-symfony's latest version 0.0.1 as it requires ext-otel_instrumentation * which is missing from your platform.

In PackageDiscoveryTrait.php line 318:
                                                                                                                                               
  Package open-telemetry/opentelemetry-auto-symfony has requirements incompatible with your PHP version, PHP extensions and Composer version:  
    - open-telemetry/opentelemetry-auto-symfony 0.0.1 requires ext-otel_instrumentation * but it is not present.
```

**Additional context**
- This issue is similar to the https://github.com/open-telemetry/opentelemetry-php/issues/1000
- [Documentation](https://opentelemetry.io/docs/instrumentation/php/automatic/) says nothing about the `minimum-stability` configuration.

**Comment by brettmc** [comment]
Many of the packages have gone GA now, so we removed the minimum-stability recommendation in the documentation. Please set your minimum stability to at least beta to be able to install the latest version.

**Comment by brettmc** [comment]
Closing, please re-open if the suggested solution does not work. Also, we're awaiting field testing and feedback on when to mark the library as 1.0.0, so please do provide some feedback if it works as expected for you (or not!)

**Comment by AustinReeseKipsu** [comment]
[Documentation](https://opentelemetry.io/docs/instrumentation/php/automatic/) continues to say nothing about the minimum-stability configuration, despite it being a common error.

**Comment by cmnemoi** [comment]
> Please set your minimum stability to at least beta to be able to install the latest version.

I don't understand how to do that. :(

**Comment by brettmc** [comment]
> I don't understand how to do that. :(

https://getcomposer.org/doc/04-schema.md#minimum-stability

TL;DR: add `"minimum-stability": "dev"` to your composer.json

**Comment by Mahfouz98** [comment]
Finally this works for me when i used specific version of 
`open-telemetry/opentelemetry-auto-symfony:dev-main`

I have used before `open-telemetry/opentelemetry-auto-symfony` and I thought that it should grap the latest version but obviously it was the First version 0.0.1
